UnityProject/Assets/Scripts/Messages/Client/mapping/ClientRequestLoadMap.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using MapSaver;
+using Messages.Client;
+using Mirror;
+using Newtonsoft.Json;
+using TileManagement;
+using UnityEngine;
+
+public class ClientRequestLoadMap : ClientMessage<ClientRequestLoadMap.NetMessage>
+{
+	public static Dictionary<PlayerInfo, List<string>> SaveDatas = new Dictionary<PlayerInfo, List<string>>();
+
+	public struct NetMessage : NetworkMessage
+	{
+		public Vector3 Offset00;
```

Comment:
what is offset00? explain why it exists

Reply:
so, offset00 is the offset that needs to be applied to the positions in the save file to make it so that if you spawned it in with this it would be at 0 ,0 , and then the offset is an additional want to make it place where you want it to place

Reply:
Like I've said in another reply, leave this inside the code inside of comments.
No one is going to search github PRs to find this very specific conversation to understand what that variable is